### PR TITLE
DOCSP-22726 Cluster-to-Cluster Sync Installation pagesl

### PR DESCRIPTION
**Pull request by kennethdyer**
[DOCSP-22726](https://jira.mongodb.org/browse/DOCSP-22726)

Staging Links:

* [Installation](https://docs-mongodbcom-staging.corp.mongodb.com/cluster-sync/docsworker-xlarge/DOCSP-22726-c2c-install/installation/)
* [Install on Linux](https://docs-mongodbcom-staging.corp.mongodb.com/cluster-sync/docsworker-xlarge/DOCSP-22726-c2c-install/installation/install-on-linux/)
* [Install on macOS](https://docs-mongodbcom-staging.corp.mongodb.com/cluster-sync/docsworker-xlarge/DOCSP-22726-c2c-install/installation/install-on-macos/)

>Note: The download link defaults to enterprise server given that we don't yet have one for C2C.

### CODE REVIEW

File: source/includes/steps-install-mongosync-macos.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+stepnum: 1
```
Comment (jmd-mongo) [inline source/includes/steps-install-mongosync-macos.yaml:1]: [suggestion]

We have started using the `.. procedure::` directive in place of the yaml step files. 

Here's an example from the upgrade docs: https://github.com/10gen/docs-mongodb-internal/pull/1128/files#diff-be567092a426709680a1b9e1e990044a15b2960e934b65f2781f501422b100aeR108 

Here's an example from the downgrade docs: https://github.com/10gen/docs-mongodb-internal/pull/1080/files#diff-a015134d4d03b46f057022780b3bd71eeb89e4a3e3adcdf32ae95e566f240973R119

File: source/includes/steps-install-mongosync-on-linux.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+level: 4
+ref: extract
+pre: |
+  For example, from a system shell, you can extract using the ``tar`` command:
+
+  .. code-block:: bash
+
+     tar -zxvf mongosync-*.tgz
+---
+title: Ensure the binary is in a directory listed in your ``PATH`` environment variable.
```
Comment (jmd-mongo) [inline source/includes/steps-install-mongosync-on-linux.yaml:34]: [question]

Should we include instructions on updating `PATH`?

**Review by jmd-mongo — COMMENTED** [review]
Thanks for the changes, @kennethdyer! I've left a few comments and suggestions. Please let me know if there's any other details I can send your way.

Thanks!
Joe

File: source/includes/fact-install-past-mongosync.rst [hunk 3]
```diff
@@ -0,0 +1,5 @@
+
+This tutorial installs MongoDB Cluster-to-Cluster Sync
```
[the remark this answers is not in the document]
Comment (kennethdyer) [reply source/includes/fact-install-past-mongosync.rst:2]: @jmd-mongo, Dave has a ticket in review to address this: [DOCSP-22838](https://jira.mongodb.org/browse/DOCSP-22838)

File: source/installation/install-on-macos.txt [hunk 4]
```diff
@@ -6,5 +6,55 @@ Install on macOS
...
+MongoDB Cluster-to-Cluster Sync {+version+} supports macOS 10.14 or later.
+
+Version Requirements
+~~~~~~~~~~~~~~~~~~~~
+
+.. include:: /includes/fact-mongodb-version.rst
+
+
+Install Cluster-to-Cluster Sync
+---------------------------------------------------
```
[the remark this answers is not in the document]
Comment (kennethdyer) [reply source/installation/install-on-macos.txt]: Fixed

File: source/includes/steps-install-mongosync-macos.yaml (hunk 1 above)
Comment (kennethdyer) [reply source/includes/steps-install-mongosync-macos.yaml]: Fixed

File: source/includes/steps-install-mongosync-on-linux.yaml (hunk 2 above)
Comment (kennethdyer) [reply source/includes/steps-install-mongosync-on-linux.yaml]: @jmd-mongo They don't on server install docs, not sure whether that's an old standard or current.

File: source/includes/run-c2c.rst [hunk 5]
```diff
@@ -0,0 +1,2 @@
+
+To run and administer ``mongosync``, see :ref:`Running Cluster to Cluster Sync`.
```
Comment (jmd-mongo) [inline source/includes/run-c2c.rst:2]: [question]

Should this be :ref:`c2c-running-mongosync` ?

File: source/installation/install-on-linux.txt [hunk 6]
```diff
@@ -6,5 +6,58 @@ Install on Linux
...
+Cluster-to-Cluster Sync is available on select platforms.
+
+Cluster-to-Cluster Version
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: /includes/fact-install-past-mongosync.rst
+
+Considerations
+--------------
+
```
[the remark this answers is not in the document]
Comment (jmd-mongo) [reply source/installation/install-on-linux.txt:34]: Curious to know if you opted to take this suggestion.

**Review by jmd-mongo — APPROVED** [review]
Thanks for the updates, @kennethdyer! Overall, this one LGTM mod nit. I've also left a few non-blocking questions.

Thanks!
Joe

**Review by davemungo — COMMENTED** [review]
@jmd-mongo  thanks for your feedback. I've made some updates based on this PR, but I've had to open a new PR to push them. Would you take a look at: https://github.com/mongodb/docs-cluster-to-cluster-sync/pull/9

Thanks!

**Comment by davemungo** [comment]
Superseded by https://github.com/mongodb/docs-cluster-to-cluster-sync/pull/9

**Pull request closed without merging**